packages/account/src/Components/account-limits/account-limits.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { FormikValues } from 'formik';
 import classNames from 'classnames';
-import { formatMoney, isDesktop, isMobile, useIsMounted, PlatformContext } from '@deriv/shared';
+import { formatMoney, isDesktop, isMobile, useIsMounted } from '@deriv/shared';
```

Comment:
Can you also remove `is_fully_authenticated` from line 45 since it is not being used anywhere!